packages/block-library/src/group/edit.js

```diff
@@ -39,7 +39,7 @@ function GroupEdit( { className, setBackgroundColor, backgroundColor } ) {
 				/>
 			</InspectorControls>
 			<div className={ classes } style={ styles }>
-				<InnerBlocks />
+				<InnerBlocks renderAppender={ () => <InnerBlocks.ButtonBlockAppender /> } />
```

Comment:
Thanks @talldan for the quick PR, in testing I wonder if the appender is hidden if the block is unselected and already contain items, what do people think? the condition could be

```
( isSelected || ! hasChildren ) && <Appender />
```

What do you think @mapk @jasmussen @mtias

Reply:
> Thanks @talldan for the quick PR, in testing I wonder if the appender should be hidden if the block is unselected and already contain items, what do people think? the condition could be
> 
> ```
> ( isSelected || ! hasChildren ) && <Appender />
> ```
> What do you think @mapk @jasmussen @mtias

This needs a decision. I'd tend to agree with @youknowriad that I was surprised by how out-of-place the button inserter feels after having inserted an "initial" block (i.e. once it's no longer empty).

Reply:
Yeah, I think it makes sense to hide it under those conditions. That helps preserve the 1:1 visual relationship to the front end. 👍

Reply:
Related: https://github.com/WordPress/gutenberg/pull/14241#discussion_r274755753

Reply:
> Yeah, I think it makes sense to hide it under those conditions. That helps preserve the 1:1 visual relationship to the front end. 👍

Hmm, I've been experimenting with this. I can't quite put a finger on it, but something about it feels not quite right. It's notably hard to follow when and why the appender is visible if the Group does not have a background assigned, or there are Group blocks within or adjacent to one another (similar observation as https://github.com/WordPress/gutenberg/pull/14943#issuecomment-482499428).

You can give it a try at: http://gutenberg.run/14966
Branch: #14966

Reply:
Great thoughts, thanks for trying that branch, Andrew, and what is this wonderful new gutenberg.run thing!? I love it. Here's what i see:

![group](https://user-images.githubusercontent.com/1204802/56111565-b7632680-5f58-11e9-9bdb-828b86ce111e.gif)

My very first instinct is that as soon as you've inserted a single block inside the group, we no longer need to use the generic appender for that block. That interface only feels necessary for the _setup state_. After that, the same interface that every other block uses — press enter or use the sibling inserter, feels sufficient to me. We already have so many many (+) buttons all over the place, we really need to be careful in adding more of them. 

What do you think?